libcontainer/container_linux.go

```diff
@@ -666,7 +665,7 @@ func (c *linuxContainer) Resume() error {
 		return err
 	}
 	if status != Paused {
-		return newGenericError(fmt.Errorf("container not paused"), ContainerNotPaused)
+		return newGenericError(errors.New("container not paused"), ContainerNotPaused)
```

Comment:
I'd also like to get rid of `genericError` as well but I guess that'll need to wait for the next PR?

Reply:
Ah that's what #3033 is for -- I got the two PRs confused.